oqs_scripts/testrun.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [ ! -f build/tool/bssl ]; then
```

Comment:
I don't think this check is necessary (if it's not found then let the test script just fail)

Reply:
Removing shell script.